interface_service/src/bucket.py

```diff
@@ -0,0 +1,43 @@
+"""
+Provides download and upload functionality
+For interfacing with MinIO buckets
+"""
+from minio import Minio
+from config import settings
+
+
+def authenticate():
+    """Authenticates to MinIO.
+
+    Returns:
+        Minio: The MinIO client object.
+    """
+
+    # Get credentials from env variables
+    return Minio(
+        settings["MINIO_ENDPOINT"],
+        access_key=settings["MINIO_ACCESS_KEY"],
+        secret_key=settings["MINIO_SECRET_KEY"],
+        # Without this, certificates are required
+        secure=False
```

Comment:
This should be configurable true/false flag from the properties - the production bucket uses TLS.